src/prop-types/ref.js

```diff
@@ -0,0 +1,5 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+  value: PropTypes.any,
```

Comment:
Hmm, wonder if this shoulda been `current`

Reply:
Definitely a follow up. 

I followed a few Google search results. And banking on runtime prop-type errors -- though, there were none.